pkg/utils/image/infos.go

```diff
@@ -76,13 +79,21 @@ func GetImageInfo(image string, cfg config.Configuration) (*ImageInfo, error) {
 	if fullImageName != image && !config.Configuration.GetEnableDefaultRegistryMutation(cfg) {
 		registry = ""
 	}
+	refWithTag = ReferenceWithTag(image, cfg)
```

Comment:
@1Shubham7 this won't work
```suggestion
	refWithTag = ReferenceWithTag(image, cfg)
```